runbotics/runbotics-scheduler/src/queue/process/process.controller.ts

```diff
@@ -43,6 +43,7 @@ export class ProcessController {
                 input,
                 user: request.user,
                 trigger: { name: TriggerEvent.MANUAL },
+                triggerData: { userEmail: request.user.email }
```

Comment:
same as above

Reply:
if you look at the parameter type of createInstantJob function, you will see that field triggerData is optional field, so when I want pass this field I don't have to use "?"